Raven.Database/Indexing/AbstractIndexingExecuter.cs

```diff
@@ -137,6 +132,78 @@ public void Execute()
             }
         }
 
+        protected static bool IsOperationCanceledException(Exception e)
+        {
+            var ae = e as AggregateException;
+            if (ae == null)
+            {
+                return e is OperationCanceledException;
+            }
+
+            foreach (var innerException in ae.Flatten().InnerExceptions)
+            {
+                if (innerException is OperationCanceledException)
```

Comment:
Nope, only if _all_ of them are OperationCancelledException can you return true. Otherwise you have real errors that you are hiding